Public/Sdk/SelfHost/BuildXL/Tools/QTest/Tool.QTestRunner.dsc

```diff
@@ -213,6 +214,45 @@ export function runQTest(args: QTestArguments): Result {
         )
     );
     const qTestLogsDir: StaticDirectory = result.getOutputDirectory(logDir);
+
+    // If code coverage is enabled, start a pip that will perform coverage file upload.

```

Comment:
>start [](start = 36, length = 5)

'schedule' would be more true :) Since running the script will not invoke, just build the graph to be executed.

Reply:
Fixed.